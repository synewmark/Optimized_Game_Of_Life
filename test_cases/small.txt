version https://git-lfs.github.com/spec/v1
oid sha256:d2a3728aae6baaf5123ee597120491653c9e348ddf7732820b623bf635fae587
size 274
