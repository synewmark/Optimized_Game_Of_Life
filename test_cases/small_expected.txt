version https://git-lfs.github.com/spec/v1
oid sha256:16941376f9a5373c2be7b70e25074bcb6fee6041e28c4a36700c1b9c0ed4ea88
size 250
